### PR TITLE
Settings XML

### PR DESCRIPTION
**Pull request by rajasgandhi**
**What this PR does**: adds xml files for settings signed out and logged in screen

**Which issue(s) this PR fixes**: https://github.com/FightPandemics/FightPandemics-android/issues/117

Fixes #

**Special notes for reviewers**: none

**Additional comments**: none

### CODE REVIEW

**Review by mayorjay — APPROVED** [review]
LGTM, although the ic_arrow_left will still be removed later during refactoring as we already have it in the Login module but it's not been merged
But @rajasgandhi kindly attach screenshots of what this looks like. Thanks

**Comment by rajasgandhi** [comment]
Hi, thanks. I did not know about ic_arrow_left, but all it is is this:
image

**Review by mayorjay — COMMENTED** [review]
@rajasgandhi Please attach screenshots of what this design looks like. Thanks

**Comment by rajasgandhi** [comment]
Hi @mayorJAY here are the designs
image

File: profile/src/main/res/layout/activity_logged_in.xml [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/colorWhite">
+
+    <ImageView
+        android:id="@+id/settingsBackArrow"
+        android:layout_width="24dp"
+        android:layout_height="24dp"
+        android:layout_marginStart="20dp"
```
Comment (carl05) [inline profile/src/main/res/layout/activity_logged_in.xml:12]: use dimens.xml atribute

File: profile/src/main/res/layout/activity_logged_in.xml [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+        android:src="@drawable/ic_arrow_left"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        style="@style/AndroidNativeH216Px"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="60dp"
+        android:layout_marginTop="54dp"
```
Comment (carl05) [inline profile/src/main/res/layout/activity_logged_in.xml:23]: use dimens.xml atribute

File: profile/src/main/res/layout/activity_logged_in.xml [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:layout_marginTop="99dp"
+        android:background="@color/color_line_color"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="484dp"
```
Comment (carl05) [inline profile/src/main/res/layout/activity_logged_in.xml:39]: use dimens.xml atribute

File: profile/src/main/res/layout/activity_logged_in.xml [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+        android:layout_marginTop="54dp"
+        android:text="@string/settings_action"
+        app:layout_constraintBottom_toBottomOf="@id/settingsBackArrow"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:layout_marginTop="99dp"
```
Comment (carl05) [inline profile/src/main/res/layout/activity_logged_in.xml:32]: use dimens.xml atribute

File: profile/src/main/res/layout/activity_logged_in.xml [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+        android:src="@drawable/ic_arrow_down"
+        app:layout_constraintBottom_toBottomOf="@id/feedbackContainer"
+        app:layout_constraintRight_toRightOf="@id/feedbackContainer"
+        app:layout_constraintTop_toTopOf="@id/feedbackContainer" />
+
+    <TextView
+        style="@style/AndroidNativeP311Px"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="74dp"
```
Comment (carl05) [inline profile/src/main/res/layout/activity_logged_in.xml:235]: use dimens.xml atribute check all files please

File: profile/src/main/res/menu/menu_profile.xml [hunk 6]
```diff
@@ -6,6 +6,7 @@
         android:id="@+id/settings"
         android:icon="@drawable/ic_settings"
         android:title="@string/settings_action"
+        android:onClick=""
```
Comment (carl05) [inline profile/src/main/res/menu/menu_profile.xml:9]: can we remove this line?

**Review by carl05 — CHANGES_REQUESTED** [review]

File: profile/src/main/res/menu/menu_profile.xml (hunk 6 above)
Comment (rajasgandhi) [reply profile/src/main/res/menu/menu_profile.xml:9]: yes we can

**Comment by carl05** [comment]
@

**Comment by carl05** [comment]
@rajasgandhi can you please update the repository with the suggestions?

**Review by carl05 — APPROVED** [review]

**Pull request merged**